### PR TITLE
[MDB IGNORE] Eliminates Toxins

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Repaths everything referring to "toxins" while actually meaning either the room in science or plasma gas. While this PR might be disrespectful to our forefathers, given this is (I believe) a holdover from as far back as the Exadv1 days, this has constantly irked me since I started working with the code. None of the player-facing stuff has referred to plasma as toxin since before 4407 hit, besides the Toxins Lab, and yet all of the type-paths are still pointing at toxins, making it a nightmare to search for in a map editor, and making the code needlessly easy to confuse with that of toxin damage. So this just fires it into the sun.

Anything relating to Toxins, the science subdepartment, now makes reference to Ordnance instead. This felt fitting enough given the focus of the subdepartment is around the creation of and testing of explosives.
Anything relating to plasma gas has, fittingly, been made to refer to plasma gas.

Edit: Ah yes, I feel I should probably apologise off the bat for the size of this PR- the code touched is mostly atmos machinery and simplemobs, a few sprites here and there, and of course the station maps + a few offstation maps.

## Why It's Good For The Game
Makes the code more legible and makes mapping less painful.

## Changelog
:cl:
code: Management would like to announce that in accordance with new company policy, the Toxins Lab will henceforth be known as the Ordnance Lab. In addition, a number of company protocols have been rewritten with this in mind.
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -259,7 +259,7 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 	greyscale_config = /datum/greyscale_config/canister
 	greyscale_colors = "#9b5d7f"
 
-/obj/machinery/portable_atmospherics/canister/toxins
+/obj/machinery/portable_atmospherics/canister/plasma
```
Comment (tralezab) [inline code/modules/atmospherics/machinery/portable/canister.dm:262]: oh my god yessssssssssssssssssssssssssssssssssssssssssssssssssssssssss

**Comment by Mothblocks** [comment]
i wonder if people will *ever* call it ordnance instead of toxins. people still call the cryptographic sequencer the emag

**Review by Mothblocks — CHANGES_REQUESTED** [review]
add a txt file for all conversions in here just like the rest

https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths


in fact you shouldve done that from the beginning wouldve saved you a ton of work

**Comment by EOBGames** [comment]
> i wonder if people will _ever_ call it ordnance instead of toxins. people still call the cryptographic sequencer the emag

Doesn't particularly concern me what people actually end up calling it- it being something less pants-on-head than toxins in the code pleases me

**Comment by Mothblocks** [comment]
yeah i figured

**Comment by tralezab** [comment]
Separating toxins damage and atmos gas is fantastic

**Comment by KittyNoodle** [comment]
Everyone will continue to call it the bomb lab or toxins lab, but the other changes will be great

**Comment by ATH1909** [comment]
oh my god yes

❤️

**Comment by LemonInTheDark** [comment]
I will have to perform 3 ritual sacrifices and a blood rite, but I suppose it is for the best

**Comment by JohnFulpWillard** [comment]
image

**Comment by Bawhoppen** [comment]
Ah, it's fun quirks like it being called toxins that adds flavor to the game. No need to remove, in my opinion.

**Comment by EOBGames** [comment]
> image
> 
> I wonder why the maptainer chose this for the name

I'll have you know I haven't used an ordnance survey map in 5 years! (They're quite high quality, though- very nice if you need to know the exact height of a random hill in the middle of the Yorkshire countryside)

**Comment by maxymax13** [comment]
> 
> 
> i wonder if people will _ever_ call it ordnance instead of toxins. people still call the cryptographic sequencer the emag
do you call doorjacks "Airlock Authentication Override Card™"s then?

**Comment by CPTANT** [comment]
Just call it the plasma lab, lol.

**Comment by LemonInTheDark** [comment]
Lemme give this a read over, and if I'm good with it I'll bug you about conflicts, cause that's just not fair

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/__DEFINES/obj_flags.dm [hunk 2]
```diff
@@ -44,7 +44,7 @@
 #define STOPSPRESSUREDAMAGE (1<<1) //SUIT and HEAD items which stop pressure damage. To stop you taking all pressure damage you must have both a suit and head item with this flag.
 #define BLOCK_GAS_SMOKE_EFFECT (1<<2) // blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY!
 #define MASKINTERNALS (1<<3) // mask allows internals
-#define GAS_FILTERING (1<<4) //mask filters toxins and other harmful gases
+#define GAS_FILTERING (1<<4) //mask filters plasma and other harmful gases
```
[the remark this answers is not in the document]
Comment (EOBGames) [reply code/__DEFINES/obj_flags.dm:47]: aye, fair- I was kinda on full find and replace mode there

**Comment by EOBGames** [comment]
> add a txt file for all conversions in here just like the rest
> 
> https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths
> 
> in fact you shouldve done that from the beginning wouldve saved you a ton of work

alright, I've made one- it's also a fantastic file for showing why this PR is necessary, so that's nice

**Comment by LemonInTheDark** [comment]
The ritual is finished, payments have been made, and there's only a 20% chance of aran showing up at my door with a hammer. Let's fucking dance

**Comment by maxymax13** [comment]
soulless

**Comment by LemonInTheDark** [comment]
Opinion noted and ignored

**Pull request merged**